code/modules/clothing/under/accessories.dm

```diff
@@ -348,12 +345,9 @@
 		SEND_SIGNAL(L, COMSIG_CLEAR_MOOD_EVENT, "fan_clown_pin")
 
 /obj/item/clothing/accessory/fan_mime_pin
-	name = "Mime Pin"
-	desc = "A pin to show off your appreciation for mimes and miming"
+	name = "\improper Mime Pin"
+	desc = "A pin to show off your appreciation for mimes and miming."
```

Comment:
```suggestion
	desc = "A pin to show off your appreciation for mimes and miming!"
```

Reply:
Your bias is showing 😝

Reply:
mimes would not use something so emotional like !